src/main/kotlin/org/rust/toml/RsTomlCompletionProvider.kt

```diff
@@ -58,6 +68,34 @@ class CargoTomlKeysCompletionProvider : CompletionProvider<CompletionParameters>
         })
     }
 
+    private fun dependencies(parent: TomlKeyValue): Collection<String> {
+        val response = try {
+            val name = parent.text.replace("IntellijIdeaRulezzz", "")
```

Comment:
Let's use `CompletionUtil.getOriginalElement(parent)?.text ?: ""` instead.

Reply:
Thanks, didn't find the way to do it in an accurate way